docs/4.2/admin-guide.md

```diff
@@ -2050,17 +2047,17 @@ chapter.
 
 In the scenario illustrated above a user would execute the following commands:
 
-```bsh
+``` bash
 # Authentication step to retrieve the certificates. tsh login places the SSH
 # certificate into ~/.tsh as usual and updates ~/.kube/config on your local
 # client with Kubernetes credentials:
-[client] $ tsh --proxy=teleport.example.com login
```

Comment:
This was originally done because we had some people who didn't understand that you were supposed to run these commands on your local client machine rather than after SSHing into a Teleport node. It's probably fine but in future we might want to make it clearer.